charts/helm_lib/README.md

```diff
@@ -0,0 +1,847 @@
+# Helm utils template definitions for Deckhouse modules
+
+## Table of contents
+
+| Table of contents |
```

Comment:
The ToC could be just a two-level nested list. I believe it is easier to read than one-level mixture of headings and links

```
- Envs For Proxy
  - [helm_lib_envs_for_proxy](#helm_lib_envs_for_proxy)
- High Availability
  - [helm_lib_is_ha_to_value](#helm_lib_is_ha_to_value)
  - [helm_lib_ha_enabled](#helm_lib_ha_enabled)
```

- Envs For Proxy
  - [helm_lib_envs_for_proxy](#helm_lib_envs_for_proxy)
- High Availability
  - [helm_lib_is_ha_to_value](#helm_lib_is_ha_to_value)
  - [helm_lib_ha_enabled](#helm_lib_ha_enabled)

Reply:
We decided keep table for first iteration.